requirements.txt

```diff
@@ -1,24 +1,8 @@
-cachetools==5.5.2
-certifi==2025.1.31
-charset-normalizer==3.4.1
+python-dotenv==1.1.0
+requests==2.32.3
 google-api-core==2.24.2
 google-api-python-client==2.165.0
 google-auth==2.38.0
 google-auth-httplib2==0.2.0
 google-auth-oauthlib==1.2.1
-googleapis-common-protos==1.69.2
-httplib2==0.22.0
-idna==3.10
-oauthlib==3.2.2
-pip==23.0.1
-proto-plus==1.26.1
-protobuf==6.30.1
-pyasn1==0.6.1
-pyasn1_modules==0.4.1
-pyparsing==3.2.3
-requests==2.32.3
-requests-oauthlib==2.0.0
-rsa==4.9
-setuptools==65.5.0
-uritemplate==4.1.1
-urllib3==2.3.0
+googleapis-common-protos==1.69.2
```

Comment:
Not sure about this change in dependencies. Which python version are you using?

Reply:
I am using python 3.12, but precisely because your intention is to make it compatible, I only left the dependencies that you are actually using directly. This allows the related dependencies to be installed according to each user's environment. You didn't realize it, but `setuptools` has an incompatibility due to the jump between version 65^ and 73^.
The problem with pip freeze is that it throws all the dependencies at you, and it is the responsibility of the developer to clean up the ones that are used directly and the ones that are not.